dapps/profile.vue/src/i18n/en.json

```
@@ -492,6 +492,8 @@
       "transactions": {
         "charged-at": "charged at",
         "credit-charged": "Credit recharged",
+        "credit-running-recharge": "Credit recharge not yet completed",
```

```suggestion
        "credit-running-recharge": "Charging credit card not completed yet",
```
1. Charge instead of Recharge
2. Aren't we charging the "wallet" and not the "credit card"? (Wallet aufladen, nicht Kreditkarte)